core/templates/domain/topic/topic-object.model.spec.ts

```diff
@@ -309,7 +309,7 @@ describe('Topic object factory', () => {
     expect(_sampleTopic).not.toBe(secondTopic);
     expect(_sampleTopic).not.toEqual(secondTopic);
 
-    _sampleTopic.copyFromTopic(secondTopic);
+    _sampleTopic = secondTopic.copyFromTopic(secondTopic);
```

Comment:
I'm not sure what this means -- the previous code made more sense. Why is it written this way?

Reply:
Done